share/picongpu/tests/openPMD-viewer-test/lib/python/test/param-test.py

```diff
@@ -0,0 +1,235 @@
+import argparse
+import sys
+import numpy as np
+import scipy.constants as spc
+from openpmd_viewer import OpenPMDTimeSeries
+import openpmd_viewer
+
+print(openpmd_viewer.__version__)
+parser = argparse.ArgumentParser(description="1")
+
+
+parser.add_argument(
+    "-r",
+    help="Path of the folder where the results"
+    " of the test-suite should be saved",
+    dest="data",
+    type=str,
+)
+
+args = parser.parse_args()
+
+# Initialize the OpenPMDTimeSeries object with the path to the data
+ts = OpenPMDTimeSeries(args.data)
+
+# Define parameters for electromagnetic fields and charge/energy densities
+field_param = {
+    "Ex": -30.0e2,
+    "Ey": -20.0e4,
+    "Ez": -10.0e6,
+    "Bx": 25,
+    "By": -50,
+    "Bz": 50,
+    "e_all_chargeDensity": -spc.elementary_charge * 1e25,
+    "e_all_energyDensity": spc.m_e
+    * spc.c**2
+    * (1.021 - 1)
+    * 1e25,
```

Comment:
could the equation be formatted nicer without causing linter issues?

Reply:
if you move the `gamma` code below in front of this dictionary definition, you could use gamma instead of the magic umber `1.021`
```suggestion
    "e_all_energyDensity": spc.m_e
    * spc.c**2
    * (gamma - 1)
    * 1e25,
```

Reply:
```suggestion
    "e_all_energyDensity": spc.m_e
    * spc.c**2
    * (1.021 - 1)
    * base_density,
```

Reply:
> could the equation be formatted nicer without causing linter issues?

what do you mean by linter issues?

Reply:
The code is quite widely spread over multiple lines. Perhaps there is a more readable and compact form that does not trigger python style issues (=linter issues).